checks.d/mysql.py

```diff
@@ -263,13 +585,24 @@ def _get_version(self, db, host):
         cursor.close()
         del cursor
         # Version might include a description e.g. 4.1.26-log.
-        # See http://dev.mysql.com/doc/refman/4.1/en/information-functions.html#function_version
+        # See
+        # http://dev.mysql.com/doc/refman/4.1/en/information-functions.html#function_version
         version = result[0].split('-')
         version = version[0].split('.')
-        self.mysql_version[host] = version
+        self.mysql_version[hostkey] = version
         self.service_metadata('version', ".".join(version))
         return version
 
+    def _collect_all_scalars(self, key, dictionary):
+        if (key not in dictionary) or (dictionary[key] is None):
```

Comment:
Haha, I just added the parentheses because I've been nitpicked about *not* using them before :smile: